region/maridia/inner-pink/Botwoon Energy Tank Room.json

```diff
@@ -1836,6 +1868,36 @@
         "Killing the Zoas with Bombs or Power Bombs will prevent them from respawning, although it is possible but tricky to dodge them with just Spring Ball."
       ]
     },
+    {
+      "link": [4, 4],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "Gravity",
+        {"refill": ["Energy"]},
+        {"or": [
```

Comment:
Disable Gravity and jump morph would be assumed possible, given Morph.
Adding requirements would be just to avoid collecting the item because you would have to drop into the sand, go under a low wall, and fight a puyo.